public/docs/advanced/truffle-with-metamask.md

```diff
@@ -1,69 +1,91 @@
-# Accessing Smart Contracts in a Browser
+# Truffle and MetaMask: Working with smart contracts in a browser
 
-To begin interacting with our smart contracts in a browser, first make sure they're compiled, deployed and that we're interacting with them via `web3` in our client-side JavaScript. We recommend using the [truffle-contract library](https://github.com/trufflesuite/truffle-contract), as it makes interacting with contracts easier and more robust.
+Before you can interact with smart contracts in a browser, make sure they're compiled, deployed, and that you're interacting with them via `web3` in client-side JavaScript. We recommend using the [truffle-contract library](https://github.com/trufflesuite/truffle-contract), as it makes interacting with contracts easier and more robust.
 
-For more information on these topics, including using `truffle-contract`, check out our [Pet Shop](/tutorials/pet-shop) or [TutorialToken](/tutorials/robust-smart-contracts-with-openzeppelin) Tutorials.
+<p class="alert alert-info">
+<strong>Note</strong>: For more information on these topics, including using `truffle-contract`, check out our [Pet Shop](/tutorials/pet-shop) or [TutorialToken](/tutorials/robust-smart-contracts-with-openzeppelin) tutorials.
+</p>
 
-# What is MetaMask?
+Once you've done the above, you're ready to use MetaMask.
 
-[MetaMask](https://metamask.io/) is the easiest way to start using Dapps in a browser. It is a Chrome extension that connects to the Ethereum network without running a full node on the browser's machine. It can connect to the Ethereum Main Network; Ropsten, Kovan and Rinkeby testnets; or a local chain such as the TestRPC. For development with Truffle this means we can use our Dapp the same way users will interact with it on a live network.
+## What is MetaMask?
 
-## Installing Metamask
+[MetaMask](https://metamask.io/) is the easiest way to interact with dapps in a browser. It is a Chrome extension that connects to an Ethereum network without running a full node on the browser's machine. It can connect to the main Ethereum network, any of the testnets (Ropsten, Kovan, and Rinkeby), or a local blockchain such as the one created by Truffle Develop or the TestRPC.
 
-Currently, MetaMask only supports Google's Chrome browser, but support for Firefox and others is planned for the future.
+![MetaMask](/docs/img/metamask.png)
 
-To install MetaMask, go to the [Chrome Web Store](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn) and click the + ADD TO CHROME button.
+For development with Truffle this means we can use our dapp the same way users will interact with it on a live network.
 
-# Using MetaMask with the TestRPC
+## Installing MetaMask
```

Comment:
Do you (should you) add a section about installing with Firefox?

Reply:
You could also simply link to the extensions and then refer people to each browser's instructions for adding extensions.

Reply:
I think it's fair to mention how to get MetaMask for both browsers, but otherwise, I'd like to keep the page browser-agnostic.

Reply:
You mention Chrome a lot in this page, which means it's browser specific. You might not be able to be browser agnostic when there are two different ways to get Metamask depending on the browser you're using.